hls4ml/converters/onnx/core.py

```diff
@@ -53,70 +45,89 @@ def parse_gemm_layer(reader, node, inputs_map, input_shapes, graph, config):
     'Softmax': 'Softmax',
     'Softsign': 'Activation',
     'Softplus': 'Activation',
-    'Clip': 'Clip',
+    # 'Clip': 'Clip',
```

Comment:
Remove commented code?

Reply:
Clip will not be hard to add, and we probably should do it, but I'll remove it for now.